api/justfile

```
@@ -64,9 +64,9 @@ _curl-get path host="localhost:50280":
 stats media="images" host="localhost:50280":
     just _curl-get "{{ media }}/stats/" {{ host }}
 
-# Launch a `pgcli` shell in the web container (requires typing credentials)
-pgcli db_host="db" db_name="openledger" db_user="deploy":
-    env DC_USER="opener" just ../exec web pgcli -h {{ db_host }} openledger
+# Launch a `pgcli` shell in the web container
+pgcli db_host="db" db_name="openledger" db_user="deploy" db_password="deploy" db_port="5432":
```

It looks like this requires `just up` first, can/should we add that as a dependency?

```suggestion
pgcli db_host="db" db_name="openledger" db_user="deploy" db_password="deploy" db_port="5432": up
```